units/src/fee.rs

```diff
@@ -344,8 +344,11 @@ mod tests {
 
     #[test]
     fn fee_wu() {
-        let fee_overflow = FeeRate::from_sat_per_kwu(10).to_fee(Weight::MAX);
-        assert!(fee_overflow.is_error());
+        let error: MathOp = FeeRate::from_sat_per_kwu(10)
```

Comment:
I'd prefer naming the variable `operation`.

Reply:
sure, no problem

Reply:
done

Reply:
Personally I think `operation_error.is_multiplication()` reads better.